tests/TestCurrencyHelper.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace PodPoint\I18n\Tests;
+
+use Illuminate\Config\Repository;
+use PHPUnit\Framework\TestCase;
+use PodPoint\I18n\CurrencyCode;
+use PodPoint\I18n\CurrencyHelper;
+
+class TestCurrencyHelper extends TestCase
```

Comment:
should be named `CurrencyHelperTest` not sure why the `TestCountryHelper` is named incorrectly

Reply:
Ah ok I'm going to rename both 👍